Apps/Admin/Client/Pages/SupportPage.razor

```diff
@@ -38,56 +43,62 @@
     </MudGrid>
 </MudForm>
 
-<HgBannerFeedback Severity="@Severity.Info" IsEnabled="HasWarning" TResetAction="MessageVerificationActions.ResetStateAction" DataTestId="banner-feedback-warning-message">
-    @MessageVerificationState.Value.WarningMessage
+<HgBannerFeedback Severity="@Severity.Info" IsEnabled="HasHasSupportUsersWarning" TResetAction="SupportUserActions.ResetStateAction" DataTestId="banner-feedback-warning-message">
```

Comment:
Typo in variable name `HasHasSupportUsersWarning` (matched in code-behind).

Reply:
Fixed.